ancar/portfolio/src/main/webapp/style.css

```diff
@@ -176,6 +162,48 @@ h3 {
     font-size: 25px;
 }
 
+#comments {
+    border-style: solid;
+    border-color: lightcoral;
+    border-width: 5px;
+    width: 50%;
+    display: flex;
+    flex-direction: row;
+}
+
+#form {
+    display: flex;
+    flex-direction: column;
+    margin-left: 20px;
+}
+
+#submit-button {
+    font-size: 20px;
+    background-color: lightcoral;
+    color: whitesmoke;
+}
+
+.buttons-comments {
+    cursor: pointer;
+    font-size: 20px;
+    background-color: lightcoral;
+    color: whitesmoke;
+    width: 200px;
+    height: 30px;
```

Comment:
these are fixed vaues again, the 200 might be wrong in a different size screen.

Reply:
Fixed in new commit ;)